app/src/main/java/co/stonephone/stonecamera/plugins/AspectRatio.kt

```diff
@@ -172,12 +170,13 @@ class AspectRatioPlugin : IPlugin {
     override val settings: List<PluginSetting> = listOf(
         PluginSetting.EnumSetting(
             key = "aspectRatio",
+            label = "Aspect Ratio",
             defaultValue = "16:9",
             options = listOf("16:9", "4:3", "FULL"),
-            render = { value ->
+            render = { value, isSelected ->
                 Text(
                     text = value,
-                    color = Color.White,
+                    color = if (isSelected) Color(0xFFFFCC00) else Color.White,
```

Comment:
Presumably hard-coded colours are good enough for the moment? Would either of these colours ever change e.g. if the user has their device set to high-contrast mode?